yetanotherco/aligned_layer
# fix: remove asserts from risc0 and sp1

# Description

- PR that removes the asserts from the sp1 and risc0 provers.

# To Test
- Run `make build_risc_zero_macos` or `make build_risc_zero_linux`.
- Run `make test_risc_zero_rust_ffi` and `make test_risc_zero_go_bindings_macos` or `make test_risc_zero_go_bindings_linux`.
- Run `make build_sp1_macos` or `make build_sp1_linux`.
- Run `make test_sp1_rust_ffi` and `make test_sp1_go_bindings_macos` or `make test_sp1_go_bindings_linux`.
- Send some risc0 and sp1 proofs and check everything is ok.
